contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -437,10 +437,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
 
         hasClaimedReward[staker][epoch] = true;
         // call fee handler to claim reward
-        require(
-            feeHandler.claimStakerReward(staker, perInPrecision, epoch),
-            "claimReward: feeHandle failed to claim"
-        );
+        feeHandler.claimStakerReward(staker, perInPrecision, epoch);
```

Comment:
May be we can change FeeHandler to:
`require(success, "staker rewards transfer failed");`
---> ` if (!success) { return false; }` and emit event

Reply:
for readability?

Reply:
Yeah mostly for readability. Not sure if we should catch this event and notify staker that this address can not receive ETH

Reply:
so idea is to add an event in feeHander.

Reply:
IMO leave it like that.. feeHandler should revert if the failing msg